### PR TITLE
Initial PR / Issue templates

### PR DESCRIPTION
**Pull request by RanbirAulakh**
**Issue #, if available:** n/a

**Description of changes:** Created a set of templates for PR and Issues

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+      label: Expected Behavior
+      description: A clear description of what you expected to happen.
+    validations:
+      required: true
+
+  - type: textarea
+    id: steps-to-reproduce
+    attributes:
+      label: Steps to Reproduce
+      description: Please explain the steps required to duplicate this issue.
```
Comment (douglas1850) [inline .github/ISSUE_TEMPLATE/bug_report.yml:48]: duplicate -> replicate

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 2]
```diff
@@ -0,0 +1,48 @@
+name: 💡 Feature Request
+description: Suggest an idea for OSML
```
Comment (douglas1850) [inline .github/ISSUE_TEMPLATE/feature_request.yml:2]: Suggest a feature for OSML

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+      label: Describe the Use Case
+      description: A clear and concise use case for what problem this feature would solve.
+    validations:
+      required: true
+
+  - type: textarea
+    id: proposed-solution
+    attributes:
+      label: Describe Preferred Solution
+      description: A clear and concise description of what you how you want this feature to be added to OSML
```
Comment (douglas1850) [inline .github/ISSUE_TEMPLATE/feature_request.yml:36]: Needs a period at the end

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+
+### Notes
+
+
+### Testing
+
+
+Before you submit a pull request, please make sure you have to following:
+
+- [ ] Your code should contain tests relevant for the problem you are solving
```
Comment (douglas1850) [inline .github/PULL_REQUEST_TEMPLATE.md:11]: Maybe reword this to: "Your code contains tests that cover all new code and changes"

**Review by douglas1850 — COMMENTED** [review]
Some minor improvements.

File: .github/ISSUE_TEMPLATE/bug_report.yml [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+      description: Please ensure you have completed all of the following.
+      options:
+        - label: I have searched for [existing issues](https://github.com/aws-solutions-library-samples/guidance-for-overhead-imagery-inference-on-aws/issues) that already report this problem, without success.
+          required: true
+
+  - type: dropdown
+    id: affected-versions
+    attributes:
+      label: OSML Branch Version
+      description: Which branch version(s) of OSML does this issue impact?
```
Comment (drduhe) [inline .github/ISSUE_TEMPLATE/bug_report.yml:20]: Which branch version(s) of OSML does this issue relate to?

File: .github/ISSUE_TEMPLATE/feature_request.yml [hunk 6]
```diff
@@ -0,0 +1,48 @@
...
+    attributes:
+      label: Describe the Use Case
+      description: A clear and concise use case for what problem this feature would solve.
+    validations:
+      required: true
+
+  - type: textarea
+    id: proposed-solution
+    attributes:
+      label: Describe Preferred Solution
```
Comment (drduhe) [inline .github/ISSUE_TEMPLATE/feature_request.yml:35]: Proposed Solution
